tests/gtest/avif_fuzztest_helpers.cc

```diff
@@ -199,7 +199,13 @@ class Environment : public ::testing::Environment {
  public:
   Environment(const char* name, const char* value)
       : name_(name), value_(value) {}
-  void SetUp() override { setenv(name_, value_, 1); }
+  void SetUp() override {
+#ifdef WIN32
```

Comment:
Please change `WIN32` to `_WIN32` . The compiler predefined macro is named `_WIN32` .